InvenTree/InvenTree/settings.py

```diff
@@ -206,6 +209,16 @@ def get_setting(environment_var, backup_val, default_value=None):
     print("ERROR: INVENTREE_MEDIA_ROOT directory is not defined")
     sys.exit(1)
 
+# Options for django-maintenance-mode : https://pypi.org/project/django-maintenance-mode/
+MAINTENANCE_MODE_STATE_FILE_PATH = os.path.join(
+    config_dir,
+    'maintenance_mode_state.txt',
+)
+
+MAINTENANCE_MODE_IGNORE_ADMIN_SITE = True
+
+MAINTENANCE_MODE_IGNORE_SUPERUSER = True
```

Comment:
@SchrodingersGat this can lead to superusers receiving errors while plugins are reloaded. In the reload process all apps are reinitialized -> might cerate inconsistencies if someone (even superusers) access parts of InvenTree that are reloaded as the models and apps do not exsist